packages/angular_devkit/schematics/tasks/node/index.ts

```diff
@@ -28,8 +29,11 @@ export class BuiltinTaskExecutor {
     name: RunSchematicName,
     create: () => import('../run-schematic/executor').then(mod => mod.default()),
   };
-  static readonly TslintFix: TaskExecutorFactory<{}> = {
-    name: TslintFixName,
-    create: () => import('../tslint-fix/executor').then(mod => mod.default()),
-  };
+  static readonly TslintFix = jobs.lazyLoadJob(() => {
```

Comment:
These are still tasks even though they are internally implemented as jobs.
Why not keep the TaskExecutorFactory and add the `input` schema option.  And maybe a temporary flag to indicate this is a job-based task.
Tasks don't return any values so there is no need for an output schema.

The registration method can then wrap the job-based tasks as appropriate.